kubernetes/configs/prod/certified_mail_sender.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+SMTP_SENDER="interopuser/interoperabilita@pec.pagopa.it"
+SMTP_ADDRESS="smtps.pec.aruba.it"
+SMTP_PORT=465
+WITH_SSL=true
```

Comment:
considering these are in a global scope, I suggest to rename them (same for all envs obv)
```suggestion
PEC_SMTP_SENDER="interopuser/interoperabilita@pec.pagopa.it"
PEC_SMTP_ADDRESS="smtps.pec.aruba.it"
PEC_SMTP_PORT=465
PEC_WITH_SSL=true
```